tests/Benchmarks/Encodings/AsciiDecoding.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+using System.Text.Utf8;
+using BenchmarkDotNet.Attributes;
+
+namespace Benchmarks.Encodings
+{
+    public class AsciiDecoding
```

Comment:
If I want to measure allocations, can I do it without changing the sources? i.e. through the "switcher"